app/assets/src/components/views/report/SampleDetailsSidebar/constants.js

```diff
@@ -57,13 +56,57 @@ export const VECTOR_METADATA_SECTIONS = [
       "nucleotide_type",
       "sample_unit",
       "collection_date",
+      "collection_lat",
+      "collection_long",
+      "collected_by"
+    ]
+  },
+  {
+    name: "Specimen Info",
+    keys: [
+      "reported_sex",
+      "comp_sex",
```

Comment:
Any reason not use `computed` (assuming thats the meaning) ? After all is the same size as reported.

Reply:
These are the keys biohub sent to me. In their template it's "reported_sex" and "comp_sex". I agree with you that it's arbitrary, but it'll make uploading their metadata a little easier.

When we move to customizable metadata, we can adjust our baseline fields to be the way we want them.